docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -42,9 +42,11 @@ The following app file types are supported for real device tests:
 
 ## Using the W3C WebDriver Specification
 
-Sauce Labs encourages adoption of the W3C WebDriver protocol, but supports both JSON Wire Protocol (JWP) or W3C in all currently supported Appium 1.X versions (Appium 2.0 will deprecate support for JWP).
+:::warning Migrating Appium Real Device Tests to W3C Protocol
+Effective April(??) 2022, for Appium real device tests, we are deprecating support for the JSON Wire Protocol (JWP) in favor of the W3C WebDriver Protocol. To avoid disruption to your testing and development cycle, we encourage you to migrate any Appium real device tests written in JWP to W3C Protocol as soon as possible. [Learn more](https://support.saucelabs.com/hc/en-us/articles/4412359870231)
```

Comment:
Hi @kimsauce,
This is incorrect, we are not deprecating JWP. We recommend and encourage customers to migrate their tests from JWP to W3C because it is the more modern standard and if they wish to use Appium 2.0 they would require W3C anyway. Thus it would save them time later, even if they still want to use Appium 1.x